### DISCUSSION TITLE
Acrylic effects for inactive windows?

### DISCUSSION THREAD

**Discussion by creewick** [AvaloniaUI/Avalonia#16055] [Q&A]
Hello! I’m wondering if it’s possible for a window to always maintain its background effects (mica, acrylic, blur, …)?
I know that the default behavior is that a window has these effects only while it is active. But is there some way to avoid losing the effect?

**Comment by StefanKoell** [comment]
Did you test 11.1.0-RC1? I'm using this version and for me the background effect is always applied (active or inactive window). I had to implement a specific style to switch Mica off when the window is not active.
